opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/FamilyOtherMemberProfileFragment.java

```diff
@@ -0,0 +1,44 @@
+package org.smartregister.reveal.fragment;
+
+import android.os.Bundle;
+import android.view.View;
+
+import org.smartregister.family.fragment.BaseFamilyOtherMemberProfileFragment;
+import org.smartregister.family.model.BaseFamilyOtherMemberProfileFragmentModel;
+import org.smartregister.family.presenter.BaseFamilyOtherMemberProfileFragmentPresenter;
+import org.smartregister.family.util.Constants;
+import org.smartregister.reveal.R;
+
+public class FamilyOtherMemberProfileFragment extends BaseFamilyOtherMemberProfileFragment {
+    public static BaseFamilyOtherMemberProfileFragment newInstance(Bundle bundle) {
+        Bundle args = bundle;
+        BaseFamilyOtherMemberProfileFragment fragment = new FamilyOtherMemberProfileFragment();
```

Comment:
Same question about losing specificity  in subclass?

Reply:
Changed to Child Fragment